bundles/CoreBundle/Resources/translations/sv.json

```diff
@@ -692,7 +692,8 @@
     "login_token_no_password_error": "Användaren har inte ställt in lösenord.",
     "targeting_toolbar": "Targeting Toolbar",
     "sort_children_by": "Sortera barn efter",
-    "by_key": "Nyckel (Alfabetiskt)",
+    "by_key": "Nyckel (A till Z)",
+    "by_key_reverse": "Nyckel (Z till A)",
```

Comment:
*A till Ö* and *Ö till A* are the colloquial Swedish phrases that indicate sort order, even if ÅÄÖ may not actually be available in the possible valid keys.